ansible/roles/operator-pipeline/templates/openshift/tasks/cert-project-check.yml

```diff
@@ -5,6 +5,7 @@ metadata:
   name: certification-project-check
 spec:
   params:
+    - name: image_tag
```

Comment:
This param isn't being used in all these tasks. Do you intend to fix that?

Reply:
sure